ci/unit_tests/wawCommons/replaceValue_test.py

```diff
@@ -0,0 +1,333 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import pytest, unittest
+
+import wawCommons
+from ...test_utils import BaseTestCaseCapture
+
+class TestMain(BaseTestCaseCapture):
+
+    sourceJsonDict = {
+        'notTargetKeyA': None,
+        'notTargetKeyB': 'notToBeReplacedValue',
+        'notTargetKeyC': 'targetKey',
+        'notTargetKeyD': {
+            'notTargetKeyDA': None,
+            'notTargetKeyDB': 'notToBeReplacedValue',
+            'notTargetKeyDC': 'targetKey',
+            'notTargetKeyDD': {
+                'notTargetKeyDDA': None,
+                'notTargetKeyDDB': 'notToBeReplacedValue',
+                'notTargetKeyDDC': 'targetKey',
+                'targetKey': {},
+                'notTargetKeyDDE': []
+            },
+            'notTargetKeyDE': [
+                None,
+                'targetKey',
+                'notTargetKeyDEC',
+                {},
+                []
+            ]
+        },
+        'targetKey': [
+            None,
+            'targetKey',
+            'notTargetKeyEC',
+            {
+                'notTargetKeyEDA': None,
+                'notTargetKeyEDB': 'notToBeReplacedValue',
+                'notTargetKeyEDC': 'targetKey',
+                'targetKey': {},
+                'notTargetKeyEDE': []
+            },
+            [
+                None,
+                'targetKey',
+                'notTargetKeyEEC',
+                {},
+                []
+            ]
+        ]
+    }
+
+    sourceJsonList = [
+        None,
+        'targetKey',
+        'notTargetKeyC',
+        {
+            'notTargetKeyDA': None,
+            'notTargetKeyDB': 'notToBeReplacedValue',
+            'notTargetKeyDC': 'targetKey',
+            'notTargetKeyDD': {
+                'notTargetKeyDDA': None,
+                'notTargetKeyDDB': 'notToBeReplacedValue',
+                'notTargetKeyDDC': 'targetKey',
+                'targetKey': {},
+                'notTargetKeyDDE': []
+            },
+            'notTargetKeyDE': [
+                None,
+                'targetKey',
+                'notTargetKeyDEC',
+                {},
+                []
+            ]
+        },
+        [
+            None,
+            'targetKey',
+            'notTargetKeyEC',
+            {
+                'notTargetKeyEDA': None,
+                'notTargetKeyEDB': 'notToBeReplacedValue',
+                'notTargetKeyEDC': 'targetKey',
+                'targetKey': {},
+                'notTargetKeyEDE': []
+            },
+            [
+                None,
+                'targetKey',
+                'notTargetKeyEEC',
+                {},
+                []
+            ]
+        ]
+    ]
+
+    def test_sourceNull(self):
+        ''' Tests if sourceJson is Null '''
+        rcReplacementNumber, rcJson = wawCommons.replaceValue(None, 'targetKey', 'replacementJson')
```

Comment:
I always thought that "rc" is abbreviation for "return code". So, I'm not sure if it makes sense in this context.

Reply:
ok